packages/core/src/signals/isDefault.ts

```diff
@@ -0,0 +1,11 @@
+import {Signal} from './SignalContext';
+
+export function isDefault<TSetterValue, TValue extends TSetterValue, TOwner>(
```

Comment:
Okay, if we're putting it in the `core` package, we might as well make it a method of `SignalContext`

Reply:
I'd argue for using a separated method, since it doesn't feel right to have normal users accessing `signal.context` methods when there can be an exposed method for doing so. Putting it on the `invokable` would also be fine.